test.js

```diff
@@ -297,3 +297,11 @@ ${dimSide}foo${dimSide}
 ${dimBottomBorder}
 	`);
 });
+
+test('text wraps when content > columns', t => {
+	const longContent = 'ab'.repeat(process.stdout.columns);
+	const lineLengths = m(longContent).split('\n').map(line => line.length);
+	for (const len of lineLengths) {
+		t.is(len, process.stdout.columns);
+	}
+});
```

Comment:
I think this feature needs a lot more tests. It's a difficult feature with a lot of potential issues.